subroutines.sh

```diff
@@ -15,7 +15,7 @@ send_command() {
   else
 	if [[ $message == *"Failed to execute command: car could not execute command"* ]]; then
 	 log_error $message
-	 log_notice "Skipping command $1"
+	 log_notice "Skipping command $@ to vin $vin"
```

Comment:
This is the most important change (fix).

Reply:
nice